tests/test_fields_email.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals, absolute_import
+
+import unittest
+
+from conformity.error import Error
+from conformity.fields import EmailAddress
+
+
+class EmailFieldTests(unittest.TestCase):
+    """
+    Tests emailaddress fields
+    """
+    # Common valid and invalid patterns excerpted from
+    # https://en.wikipedia.org/wiki/Email_address
+    valid_emails = [
```

Comment:
I think we should just stick to Django compatibility. Probably should port the tests from Django for completeness and comparison.

Reply:
Django has been considering just moving to `"@" in email` as our validator for years, but there's not been enough momentum around UTF-8 based email addresses (rather than, say, punycode based ones which still allow for internationalised domain names) that it's happened yet. I'd say we're fine with this for now considering the rest of our system would likely choke on a full utf-8 email address.

Reply:
Agreed.